.github/workflows/release.yml

```diff
@@ -70,9 +70,13 @@ jobs:
         export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
         export AKAMAI_CHE_AUTH=${{ secrets.AKAMAI_CHE_AUTH }}
         ./make-release.sh --version ${{ github.event.inputs.version }}
+    - name: Clean image cache
```

Comment:
hello, I think it won't remove anything

Reply:
so you'd suggest we do `prune -f` instead ?